parts/kubernetesmastercustomscript.sh

```diff
@@ -103,32 +109,75 @@ function setDockerOpts () {
     sed -i "s#^DOCKER_OPTS=.*#DOCKER_OPTS=${1}#" /etc/default/kubelet
 }
 
-function configNetworkPolicy() {
+function configAzureNetworkPolicy() {
+	CNI_VERSION=v0.4.0
+	AZURE_PLUGIN_VERSION=v0.7
+
+    # Create network config file.
+    CNI_CONFIG_DIR=/etc/cni/net.d
+    NET_CONFIG_FILE=$CNI_CONFIG_DIR/10-azure.conf
+    mkdir -p $CNI_CONFIG_DIR
+
+    cat <<- EOF > $NET_CONFIG_FILE
+	{
+	  "cniVersion": "0.2.0",
```

Comment:
I have no control over "cniVersion" in the netconfig file. This is a well-known field defined in CNI spec.

I can change CNI_VERSION. In retrospect, CNI_RELEASE_VERSION would probably have been a better name. I will move this function to a common shell script when adding support for DC/OS in the next week or so. Do you mind if I rename this at that time? Would save us the time to retest the change.